topics/ecology/tutorials/champs-blocs/tutorial_FR.md

```diff
@@ -0,0 +1,200 @@
+> Vous pouvez revenir à tout moment là où vous avez laissé le didacticiel en cliquant sur {% icon level %}.
+>
+> > <hands-on-title>Log in to Galaxy</hands-on-title>
+> > 1. Ouvrez votre navigateur internet préféré (Chrome, Safari ou Firefox, s'il vous plaît, pas Edge ;) !!!)
+> > 2. Rendez-vous sur votre instance Galaxy
+> > 3. Se connecter ou s'enregistrer
+> >
+> > ![Capture d'écran de l'interface Galaxy avec le bouton d'enregistrement ou de connexion en surbrillance](../../images/champbloc/galaxy_homepage.png "Galaxy homepage")
+> >
+> > Cette capture d'écran présente l'instance Galaxy Ecology, accessible à l'adresse [usegalaxy.eu](https://ecology.usegalaxy.eu/)
+> >
+> {: .hands_on}
+>
+> La page d'acceuil de Galaxy est divisée en 3 parties:
+> * Les outils sur la gauche
+> * La panneau de visualisation au milieu
+> * L'historique des analyses et l'ensemble des fichiers utilisés et générés sur la droite
+>
+> ![Capture d'écran de l'interface Galaxy, le panneau des outils est à gauche, le panneau principal est au centre et l'historique est à droite.](../../images/champbloc/galaxy_interface.png "Galaxy interface explanation")
+>
+> La première fois que vous utiliserez Galaxy, il n'y aura aucun fichier dans votre panneau d'historique.
+{: .details}
+
+Concentrons-nous maintenant sur notre workflows d'analyse sur l'état écologique de champ de blocs
+
+![workflows d'analyse sur l'état écologique du champ de blocs](../../images/champbloc/workflow.png "Workflow")
+
+# Centraliser les données
+
+{% include _includes/cyoa-choices.html option1="Oui" option2="Non" default="Oui" text="Are your ESTAMP data ready ?" %}
+
+<div class="Non" markdown="1">
+> 1. Télécharger les données depuis la base ESTAMP [estamp.afbiodiversite.fr](https://estamp.afbiodiversite.fr/donnees). Vous obtenez une archive zip.
+>
+> 2. Dézipper le dossier. Dans le dossier, il y a 3 fichiers .csv qui vont nous intéresser :
+> - champbloc_ivr.csv
+> - champbloc_qecb.csv
+> - ficheterrain.csv
+</div>
+
+> <hands-on-title>Télerversement des données dans Galaxy</hands-on-title>
+> 1. Importer les données dans Galaxy
+>
+>  * Ouvrir l'outil Galaxy de téléversement de données {% icon galaxy-upload %}
+>  * Sélectionner **Choose local files**
+>    ![Choose local files](../../images/champbloc/upload.png "upload your data")
+>  * Naviguez dans votre ordinateur et récupérez vos fichiers ESTAMP (sélectionnez les trois fichiers mentionnés précédemment : champbloc_ivr.csv, champbloc_qecb.csv et ficheterrain.csv)
+>
+>  * Cliquer sur **Start**
+>    ![Start charging your data](../../images/champbloc/start.png "Start")
+>  * Cliquer sur **Close**
+>    ![Close upload your data](../../images/champbloc/close.png "Close")
+>  Vous devez attendre que les données de votre historique s'affichent en couleur verte sur le panneau de droite avant de pouvoir les utiliser.
+>    ![Data ready](../../images/champbloc/after_before.png "Data ready")
```

Comment:
all of these figure captions ("Start", "Close") and the alt-text here shows in english

Reply:
ok. @Marie59 do you think you can propose french version of the images ? ;)